### DISCUSSION TITLE
Deeplink syntax appears broken for New Codespace from Template

### DISCUSSION THREAD

**Discussion by aaronsteers** [community/community#53816] [Codespaces] [answered]
### Select Topic Area

Bug

### Body

It looks like our old syntax for creating a new Codespace from a repo template has broken.

We were using this syntax:

[codespaces/new?template_repository=meltano/meltano-codespace-ready](https://github.com/codespaces/new?template_repository=meltano/meltano-codespace-ready)

The above doesn't seem to understand templates anymore and instead will prompt the user to select a repo to use for the new codespace. Which also is the same URL posted to the top of this post. (From what I can tell, this clones the existing repo rather than starting a new repo from the template.)

Is there a new syntax we should use?

cc @joshspicer 

Related to:

- [Custom landing page to select Codespace template from multiple · community · Discussion #44949](https://github.com/orgs/community/discussions/44949)

**Comment by bdmac** [comment, marked as answer]
I'm sorry about that! We didn't realize folks were using that parameter still. We'll get support added back but in the meantime you can just use the fancy new URL like https://github.com/codespaces/new/meltano/meltano-codespace-ready!

**Reply by aaronsteers** [reply]
Thanks, @bdmac !
